docs/en/operations/tips.md

```diff
@@ -111,6 +111,12 @@ On newer Linux kernels transparent huge pages are alright.
 $ echo 'madvise' | sudo tee /sys/kernel/mm/transparent_hugepage/enabled
 ```
 
+If you want to modify the transparent huge pages setting permanently, editing the `/etc/default/grub` to add the `transparent_hugepage=never` to the `GRUB_CMDLINE_LINUX_DEFAULT` option:
+
+```bash
+$ GRUB_CMDLINE_LINUX_DEFAULT="transparent_hugepage=madvise quiet splash"
+```
+
```

Comment:
```suggestion
If you want to modify the transparent huge pages setting permanently, edit the `/etc/default/grub` to add the `transparent_hugepage=never` to the `GRUB_CMDLINE_LINUX_DEFAULT` option:

```bash
$ GRUB_CMDLINE_LINUX_DEFAULT="transparent_hugepage=madvise ..."
```

```

Reply:
this [answer](https://stackoverflow.com/a/52591781/4395754) suggest that you also need to run `update-grub` command.

Reply:
Updated. Thanks :).